### ISSUE TITLE
Type 'MsgReader.Outlook.Storage.Message' is not defined.

### ISSUE THREAD

**Issue by NickJWilliams** [Sicos1977/MSGReader#80]
Hi, 

Updating to the latest version in NuGet (v2.1.0) I am getting the below issue in my VB.NET project.

> Type 'MsgReader.Outlook.Storage.Message' is not defined.	Scripting	

Version 2.0.14 works fine.

Thanks
Nick

**Comment by XplittR** [comment]
Possibly related to #74 
Can you check the changes that happened to your project files when you updated to the v2.1.0, and see if the same behavior happens in that happened to the C#-project?

**Comment by NickJWilliams** [comment]
Hi,

I've only just discovered MsgReader, I added the latest version to an empty test project with no joy. Downgraded to v2.0.14 and it worked. Upgraded again, namespace does not exist. 
Seems to be the same in both VB.NET and C#, although VB.NET is my preferred language.

I guess I could just add the DLL?

Thanks
Nick

**Comment by Sicos1977** [comment]
There is a bug in the latest version that I still have to fix. This is also why I hide the 2.1.0 package on nuGet. 

It try to fix it this evening (morning now over here)

**Comment by NickJWilliams** [comment]
Thanks

**Comment by igitur** [comment]
But v2.1.0 is NOT hidden on NuGet.

**Comment by Sicos1977** [comment]
Oops... I thought that I did hide it.... now it is.

**Comment by Sicos1977** [comment]
I published a new nuGet package that worked OK for me.

> PM> Install-Package MSGReader -Version 2.1.1


Attempting to gather dependency information for package 'MSGReader.2.1.1' with respect to project 'DontNetNukeExtractor', targeting '.NETFramework,Version=v4.6.1'
Gathering dependency information took 501,06 ms
Attempting to resolve dependencies for package 'MSGReader.2.1.1' with DependencyBehavior 'Lowest'
Resolving dependency information took 0 ms
Resolving actions to install package 'MSGReader.2.1.1'
Resolved actions to install package 'MSGReader.2.1.1'
Retrieving package 'MSGReader 2.1.1' from 'nuget.org'.
  GET https://api.nuget.org/v3-flatcontainer/msgreader/2.1.1/msgreader.2.1.1.nupkg
  OK https://api.nuget.org/v3-flatcontainer/msgreader/2.1.1/msgreader.2.1.1.nupkg 1045ms
Installing MSGReader 2.1.1.
Adding package 'MSGReader.2.1.1' to folder 'C:\Users\Kees\Documents\Visual Studio 2017\Projects\DontNetNukeExtractor\packages'
Added package 'MSGReader.2.1.1' to folder 'C:\Users\Kees\Documents\Visual Studio 2017\Projects\DontNetNukeExtractor\packages'
Added package 'MSGReader.2.1.1' to 'packages.config'
Successfully installed 'MSGReader 2.1.1' to DontNetNukeExtractor
Executing nuget actions took 2,94 sec
Time Elapsed: 00:00:03.4522812
PM>
